python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -94,25 +88,50 @@ def __init__(self, is_session_renewal, hostname="", login="", fixed_host=False,
             self.ui.site.setSelection(8, 8)
             self.ui.site.setFocus(QtCore.Qt.OtherFocusReason)
 
+        if self._is_session_renewal:
+            self._set_login_message("Your session has expired. Please enter your password.")
+        else:
+            self._set_login_message("Please enter your credentials.")
+
+        # Select the right first page.
+        self.ui.stackedWidget.setCurrentWidget(self.ui.login_page)
+
         # hook up signals
         self.ui.sign_in.clicked.connect(self._ok_pressed)
-        self.ui.cancel.clicked.connect(self.reject)
```

Comment:
The clicked() signal is connected to the reject() slot on the dialog in QDesigner..